cmd/miner_client/src/cpu_solver.rs

```diff
@@ -55,11 +55,11 @@ impl Solver for CpuSolver {
         nonce_tx: mpsc::UnboundedSender<SealEvent>,
         mut stop_rx: mpsc::UnboundedReceiver<bool>,
     ) {
-        let thread_num = self.config.miner_thread();
-        let worker_txs = (0..thread_num)
+        let thread = self.config.miner_thread();
```

Comment:
I think using thread confused rust Thread

Reply:
How about changing it to `thread_counter`

Reply:
i think is better